internal/controller/numaflowcontrollerrollout_controller.go

```diff
@@ -64,6 +65,11 @@ const (
 const (
 	ControllerNumaflowControllerRollout = "numaflow-controller-rollout-controller"
 	NumaflowControllerDeploymentName    = "numaflow-controller"
+	DefaultNumaflowControllerImageName  = "numaflow"
+)
+
+var (
+	delayedRequeue = ctrl.Result{RequeueAfter: 20 * time.Second}
```

Comment:
why we only have a `delayedRequeue` specifically to numaflow controller but not isbsvc or pipeline?

Reply:
sorry, I'm also using it in isbsvc - let me move this to common.go

Reply:
We're waiting for the Pipelines to get paused - we explicitly want to delay before we check again